UI/src/view/MassTransitStopForm.js

```diff
@@ -404,19 +404,38 @@
 
         return _.sortBy(properties, function(property) {
           return _.indexOf(propertyOrdering, property.publicId);
+        }).filter(function(property){
+          return _.indexOf(propertyOrdering, property.publicId) >= 0;
         });
       };
 
       var floatingStatus = function(selectedAssetModel) {
+        var text;
+        switch (selectedMassTransitStopModel.getFloatingReason()){
```

Comment:
Is it on purpose that cases 2, 4 and 5 don't have any text? Or is it getting default then?